pysensors/optimizers/_gqr.py

```diff
@@ -48,16 +48,14 @@ def __init__(self):#,idx_constrained,n_sensors,n_const_sensors,all_sensors,const
             exact_n_const_sensors : The number of sensors in the constrained region should be exactly equal to n_const_sensors.
```

Comment:
all_sensors : np.ndarray, shape [n_features]
            Optimally placed list of sensors obtained from QR pivoting algorithm.

'y' was missing from optimally